### DISCUSSION TITLE
Link maintenance settings from device model/device type to inventory items

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#45] [Ideas]
I noticed that when adding a new inventory item the Maintenance tasks and frequency are taken as set in the Device Type. But when changing the frequency in the Device Type it does not update the frequency for the inventory items of this Device Type. It would be helpful to have this linked, as sometimes you revise or add maintenance tasks to  a Device Type and I would expect all children from this parent to be updated as well.

**Comment by YvesR** [comment]
There is no reference here and can not be. 
One approach could be, that we select n-inventories and do a mass-change value for some fields.

**Reply by BioMedGPH** [reply]
Why is that? If you change the device type info like name/model/attached documents it will reflect in the individual inventories of that device type. Why not for the maintenance intervals?

**Reply by YvesR** [reply]
The service intervals are recommendations. So once set for an inventory the user may change those, and also add, remove or modify, that is the reason there is no reference.
